app/assets/javascripts/modules/FieldToggleVisibility.js

```diff
@@ -64,7 +62,23 @@ define(['jquery', 'DoughBaseComponent'], function($, DoughBaseComponent) {
     this._initialisedSuccess(initialised);
 
     return this;
-  }
+  };
+
+  FieldToggleVisibilityProto.hideRelevantTargets = function() {
+    var self = this;
+
+    this.$triggers.filter('[data-dough-field-trigger-type="show"]').each(function() {
+      var $trigger = $(this),
+        target = $trigger.attr('data-dough-field-trigger'),
+        $target = self.$targets.filter('[data-dough-field-target="' + target + '"]');
+
+      if (!$trigger.is(':checked')) {
```

Comment:
In fact it might even be easier changing line 70 to:

```
this.$triggers.filter('[data-dough-field-trigger-type="show"]:not(:checked)').each(function() {
```
and then removing this if statement completely.  What dya reckon?